### ISSUE TITLE
Please escape > character

### ISSUE THREAD

**Issue by hajovonta** [RittmanMead/md_to_conf#20]
When your markdown contains > characters, the page update will fail with 400 Bad Request.
Only the closing ( '>' ) needs to be escaped. I recommend replacing it with '\>' before the conversion.

**Comment by hajovonta** [comment]
Actually the problem seems to be more complex than I originally thought. anything that is beginning with < is interpreted as a tag, it's also true for >. Escaping them introduces various other problems. We ended up with replacing them: '<' with '< ' before the conversion, this way the converter does not interpret them as a tag.
Also, we have problems with code blocks between ''' which is standard markdown.

Good, robust and simple software otherwise.

**Comment by theBrownBug** [comment]
Yup, I am facing the same issue. @hajovonta, did you manage to overcome this?

**Comment by hajovonta** [comment]
hi @theBrownBug 

We are using it since then, but it can only handle the most simple formatting. I.e. nested lists and advanced titles are no-go.
we've modified the chunk between #787-#789 to this:

```
    with codecs.open(MARKDOWN_FILE, 'r', 'utf-8') as mdfile:
        modified_md = mdfile.read()
        for match in re.finditer(r'(\[//\]:#.)\((.*)\)', modified_md):
            md_meta = match.group(2).split(':', 1)
            meta_dict[md_meta[0]].append(md_meta[1])
        modified_md = re.sub(r'<(?=[^!])', '< ', modified_md)
        modified_md = re.sub(r'(?<=[^-])>', ' >', modified_md)

        html = markdown.markdown(modified_md,
                                 extensions=['markdown.extensions.tables',
                                             'markdown.extensions.fenced_code'])

```

this solved the problem somewhat, but I think it needs more advanced semantic parsing to completely solve the issue.
